Icon-Legend.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<!-- 
+    website Refrence for icons 
+    https://www.wta.org/go-hiking/hikes/surprise-lake-1 -->
+
+    <div class="legend">
+
+
+    <table >
+        <col  style="background-color:rgb(0, 255, 213)">
+        <col style="background-color:rgb(172, 158, 255)">
```

Comment:
Will these colors fit with the color scheme for the overall site?  Maybe we should establish a color scheme for our site.

Reply:
ohh, yeah, I just set this up for example. but yes we should decide on a color scheme.

Reply:
> 
> 
> Will these colors fit with the color scheme for the overall site? Maybe we should establish a color scheme for our site.

ohh, yeah, I just set this up for example. but yes we should decide on a color scheme.